tests/functional/test_examples.py

```diff
@@ -19,21 +19,30 @@
 from subprocess import run
 from sys import excecutable
 
+EVENT = './palace-event.py'
+HRTF = './palace-hrtf.py'
+INFO = './palace-info.py'
+LATENCY = './palace-latency.py'
+REVERB = '/palace-reverb.py'
+STDEC = './palace-stdec.py'
+TONEGEN = '/palace-tonegen.py'
```

Comment:
As `join(basename(...` please, or even better: `join(abspath(basename(...` (same for the audios).

Reply:
Why do we have to do this? What are we joining the path with?

As for the audios, you mean the ones defined in `tests/conftest.py`? Isn't it properly path-ed yet?

Reply:
See the other review below.  IMHO absolute path is a bit safer, could be wrong tho.